pymagicc/definitions/__init__.py

```diff
@@ -15,3 +15,54 @@
 # TODO: do read ins for these too
 emissions_units = {}
 concentrations_units = {}
+
+magicc6_emms_code_all_emissions = [
```

Comment:
A comment on what these are would be helpful ... Code 0, 1, PRN are not obvious to non Magicc core devs i guess

Reply:
If I put them in a function will they get picked up by the docs so these defintions appear in the docs too (which is really where they belong)?

Reply:
Good point, one could document them in a separate page in the docs or we switch to Sphinx or something else which can document module constants ...